var/spack/repos/builtin/packages/eospac/package.py

```diff
@@ -42,4 +42,18 @@ def url_for_version(self, version):
 
     def install(self, spec, prefix):
         with working_dir('Source'):
-            make('prefix=%s' % self.spec.prefix, 'install')
+            targets = ['install']
+            targets.append('CC={0}'.format(self.compiler.cc))
+            targets.append('CXX={0}'.format(self.compiler.cxx))
+            targets.append('F77={0}'.format(self.compiler.f77))
+            targets.append('F90={0}'.format(self.compiler.fc))
```

Comment:
You don't want this. This will circumvent Spack's compiler wrappers. You want to use `spack_cc` and friends instead. We already set these environment variables, but I guess the Makefile doesn't use them?

Reply:
Yes!